run_test.sh

```diff
@@ -80,14 +87,23 @@ function run_npm_build () {
 function run_py_test {
   setup_python
   cd server
-  export FLASK_ENV=test
-  python3 -m pytest tests/ -s --ignore=sustainability
+  # export FLASK_ENV=test
+  # python3 -m pytest tests/ -s --ignore=sustainability
+
   # TODO(beets): add tests for other private dc instances
   # export FLASK_ENV=test-sustainability
   # python3 -m pytest tests/sustainability/**.py
+
+  # Also test the nl_server/
+  cd ..
+  setup_python_nl_server
+  export FLASK_ENV=test
+  cd nl_server
+  python3 -m pytest tests/ -s --ignore=sustainability
```

Comment:
no need to ignore sustainability; also i don't think FLASK_ENV=test matters here.

Reply:
Done, and also uncommented a bunch of other tests which I had removed for local testing but should now be back.